lerna.json

```diff
@@ -0,0 +1,8 @@
+{
+  "version": "0.4.2",
```

Comment:
We should discuss what's the best first version for these packages.

Reply:
For master, I think it's fine to keep the same as the community. As soon as we have a position for productization, we will need to branch/tag something and we change those properties accordingly.

Reply:
/cc @porcelli